src/backend/api/Fusion.Resources.Domain/Models/QueryResourceAllocationRequest.cs

```diff
@@ -135,6 +135,8 @@ public QueryResourceAllocationRequest(DbResourceAllocationRequest entity, QueryW
         public List<QueryRequestAction>? Actions { get; set; }
         public List<QueryConversationMessage>? Conversation { get; set; }
 
+        public bool IsOrgInstanceDeleted => OrgPositionInstanceId.HasValue && OrgPositionInstance == null;
```

Comment:
Are we sure OrgPositionInstance allways will be present? 
I was thinking more of hooking in to the event stream from org and update the db with a property when an instance is deleted. 
That way we can crawl for corrupt data and use it in queries etc..